packages/x-components/src/x-modules/queries-preview/store/module.ts

```diff
@@ -22,6 +22,9 @@ export const queriesPreviewXStoreModule: QueriesPreviewXStoreModule = {
     clearQueryPreview(state, query) {
       Vue.delete(state.queriesPreview, query);
     },
+    removeQueriesPreview(state) {
+      state.queriesPreview = {};
```

Comment:
I would say that we need to remove all. The re-rendering occurs because we load already loaded queries in the state. If we only delete one (the first), then de "duplicated info" happens in the second carousel (it appeared twice). 
Well, that's what I have observed.